File: KPP1_2/src/main/java/com/university/kpp/entities/Career.java
package com.university.kpp.entities;

import java.io.Serial;
import java.io.Serializable;

public class Career implements Serializable {
    private String startDate;
    private String endDate;
    private String position;
    private String department;

    @Serial
    private static final long serialVersionUID = 412L;

    public Career(String startDate, String endDate, String position, String department) {
        this.startDate = startDate;
        this.endDate = endDate;
        this.position = position;
        this.department = department;
    }

    public String getStartDate() {
        return startDate;
    }

    public void setStartDate(String startDate) {
        this.startDate = startDate;
    }

    public String getEndDate() {
        return endDate;
    }

    public void setEndDate(String endDate) {
        this.endDate = endDate;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getDepartment() {
        return department;
    }

    public void setDepartment(String department) {
        this.department = department;
    }
}

File: KPP_lab_1_2/KPP_lab_1_2/Career.cs
namespace KPP_lab_1_2
{
    [Serializable]
    internal class Career
    {
        private DateTime start_date;
        private DateTime end_date;
        private string position;
        private string department;

        public Career(DateTime start_date, DateTime end_date, string position, string department)
        {
            this.start_date = start_date;
            this.end_date = end_date;
            this.position = position;
            this.department = department;
        }

        public DateTime Start_date
        {
            get { return start_date; }
            set { start_date = value; }
        }

        public DateTime End_date
        {
            get { return end_date; }
            set { end_date = value; }
        }

        public string Position
        {
            get { return position; }
            set { position = value; }
        }

        public string Department
        {
            get { return department; }
            set { department = value; }
        }
    }
}
